qemu/tests/cfg/chardev_tls_encryption.cfg

```diff
@@ -0,0 +1,30 @@
+- chardev_tls_encryption:
+    type = chardev_tls_encryption
+    cert_dir = /etc/pki/qemu
+    clean_cmd = rm ${cert_dir} -rf
+    start_vm = no
+    serials = ""
+    variants:
+        - host_to_guest:
+            gnutls_cmd_server = "cd ${cert_dir} &&"
+            gnutls_cmd_server += " gnutls-serv --echo --x509cafile ca-cert.pem --x509keyfile server-key.pem --x509certfile server-cert.pem -p %s"
+            extra_params = " -object tls-creds-x509,id=tls0,dir=${cert_dir},endpoint=client"
+            extra_params += " -chardev socket,id=tls_chardev,host=%s,port=%s,tls-creds=tls0"
+            extra_params += " -device isa-serial,chardev=tls_chardev,id=tls_serial"
+        - guest_to_host:
+            gnutls_cmd_client = "cd ${cert_dir} &&"
+            gnutls_cmd_client += " gnutls-cli --priority=NORMAL -p %s --x509cafile=ca-cert.pem %s --x509certfile=client-cert.pem --x509keyfile=client-key.pem"
+            extra_params = " -object tls-creds-x509,id=tls0,dir=${cert_dir},endpoint=server"
+            extra_params += " -chardev socket,id=tls_chardev,host=%s,port=%s,tls-creds=tls0,server,nowait"
+            extra_params += " -device isa-serial,chardev=tls_chardev,id=tls_serial"
+        - guest_to_guest:
+            only Linux
+            vms = 'vm1 vm2'
+            image_snapshot = yes
+            guest_cmd = "cat /dev/ttyS0 > /root/client-log &"
```

Comment:
`/root/client-log` was used not only one time,  let's use the same method as `cert_dir`

Reply:
thanks and update.

Reply:
`guest_cmd` was missed to update ~

Reply:
> `guest_cmd` was missed to update ~

Thanks and update.

Test PASS.
 (1/3) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.1.x86_64.io-github-autotest-qemu.chardev_tls_encryption.host_to_guest: PASS (20.23 s)
 (2/3) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.1.x86_64.io-github-autotest-qemu.chardev_tls_encryption.guest_to_host: PASS (20.13 s)
 (3/3) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.1.x86_64.io-github-autotest-qemu.chardev_tls_encryption.guest_to_guest: PASS (69.96 s)